tests/bluetooth/bsim/audio/src/tbs_client_test.c

```diff
@@ -95,22 +103,128 @@ static void tbs_client_read_ccid_cb(struct bt_conn *conn, int err,
+	printk("Instance: %u\n", inst_index);
+	printk("Call index: %u\n", call_index);
+}
+
+static void tbs_client_retrieve_call_cb(struct bt_conn *conn, int err,
+				    uint8_t inst_index,
+				    uint8_t call_index)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client retrieve call error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client retrieve call callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Call index: %u\n", call_index);
+}
+
+static void tbs_client_technology_cb(struct bt_conn *conn, int err,
+				    uint8_t inst_index,
+				    uint32_t value)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client bearer technology error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client bearer technology callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Technology: %u\n", value);
+
+	SET_FLAG(technology);
+}
+
+static void tbs_client_signal_strength_cb(struct bt_conn *conn, int err,
+					  uint8_t inst_index,
+					  uint32_t value)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client signal strength error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client signal strength callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Signal strength: %u\n", value);
+
+	SET_FLAG(signal_strength);
+}
+
+static void tbs_client_signal_interval_cb(struct bt_conn *conn, int err,
+					  uint8_t inst_index,
+					  uint32_t value)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client signal interval error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client signal interval callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Signal interval: %u\n", value);
+
+	SET_FLAG(signal_interval);
+}
+
+static void tbs_client_status_flags_cb(struct bt_conn *conn, int err,
+				       uint8_t inst_index,
+				       uint32_t value)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Status flags error: (%d)\n", err);
+		return;
+	}
+
+	printk("Status flags callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Status flags: %u\n", value);
+
+	SET_FLAG(status_flags);
+}
+
 static const struct bt_tbs_client_cb tbs_client_cbs = {
 	.discover = tbs_client_discover_cb,
-	.originate_call = NULL,
+	.originate_call = tbs_client_originate_call_cb,
 	.terminate_call = NULL,
-	.hold_call = NULL,
+	.hold_call = tbs_client_hold_call_cb,
 	.accept_call = NULL,
-	.retrieve_call = NULL,
+	.retrieve_call = tbs_client_retrieve_call_cb,
 	.bearer_provider_name = tbs_client_read_bearer_provider_name,
 	.bearer_uci = NULL,
-	.technology = NULL,
+	.technology = tbs_client_technology_cb,
 	.uri_list = NULL,
-	.signal_strength = NULL,
-	.signal_interval = NULL,
+	.signal_strength = tbs_client_signal_strength_cb,
+	.signal_interval = tbs_client_signal_interval_cb,
 	.current_calls = NULL,
 	.ccid = tbs_client_read_ccid_cb,
-	.status_flags = NULL,
+	.status_flags = tbs_client_status_flags_cb,
 	.call_uri = NULL,
 	.call_state = tbs_client_call_states_cb,
 	.termination_reason = NULL
```

Comment:
How come some of these are still `NULL`? Are we missing APIs to exercise these operations, or have they just not been added by this PR?